{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/dev.py

```diff
@@ -20,7 +20,8 @@ def INTERNAL_IPS(self):
         return (self.get_addr(),)
 
     # devserver must be ahead of django.contrib.staticfiles
-    INSTALLED_APPS = ('devserver',) + Common.INSTALLED_APPS + ('debug_toolbar',)
+    #INSTALLED_APPS = ('devserver',) + Common.INSTALLED_APPS + ('debug_toolbar',)
+    INSTALLED_APPS = Common.INSTALLED_APPS + ('debug_toolbar',)
```

Comment:
Why do you have removed {{devserver}} here? It is still part of the requirements.